cime_config/acme/config_files.xml

```diff
@@ -117,6 +117,23 @@
     <desc>file containing specification of all pe-layouts for primary component (for documentation only - DO NOT EDIT)</desc>
   </entry>
 
+  <entry id="SYSTEM_TESTS_DIR">
```

Comment:
No because it's inconsistent with having everything else have a path here.  For example the 
SRCROOT/components/*   path itself is defined in this file.